### ISSUE TITLE
Add memo reminder for UP changes post-verification

### ISSUE THREAD

**Issue by robertsamm** [episphere/connect#772]
Please add this memo to the top of the participant details page to remind all staff that they should only be making edits to the user profile fields post-verification for updates we want to retain, and not to be used for data corrections. This should be displayed to all users: DCEG, NORC, and sites. Since this memo is only needed for post-verification changes, suggest displaying only if participant's verification status=verified or verification status ne not yet verified or outreach timed out (if possible). Suggest adding under "back to search" button but above "Participant Details". This is a lower priority item, does not need to be completed before prod push.

Memo:
Post-verification edits: Only for actual changes (e.g. name change due to marital status, mailing address change due to move). For corrections that were not made pre-verification, follow the Data Corrections SOP.
![image](https://github.com/episphere/dashboard/assets/67074735/d54ab923-5f9f-4337-ad0f-1ad1f15a8b7f)

**Comment by robertsamm** [comment]
No longer needed, will be sure that the Data Corrections SOP / User Profile Updates SOP captures this info in sufficient detail for sites to follow.
